src/components/implementation/tests/micro_mbox_server/mbox_server.c

```diff
@@ -68,8 +68,8 @@ void cos_init(void *arg)
 			if ((int)cb1<0) evt_wait(cos_spd_id(), evt2);
 			else            break;
 		}
-		buf = cbufp2buf(cb1,sz);
-		cbufp_deref(cb1);
+		buf = cbuf2buf(cb1,sz);
+		cbuf_free(cb1);
```

Comment:
I'm interested in knowing why in the `mbox`, you have this pattern.  Why is the cb1 not used at all?